### PR TITLE
Fixes the Issue #106 by adding some minor code changes

### LINKED ISSUE #106: Some creator links are opening the extension in the new tab
**Describe the bug**
When we click on the creator link in the information popup, it opens the creator profile on a website (like flickr ) in a new tab. But some creator links instead opens the extension in a browser tab. 
In the API response, not every image has a creator url associated with it. For those instances, the creator url is set to "#". Instead, the creator name should be rendered as plain text and not a url.

**To Reproduce**
1. Search for "hello"
2. Click on the very first image.
3. In the info tab, click on the creator name.

**Expected behavior**
Those images which do not have the creator link should render the creator name as plain text.

**Screenshots**

ezgif com-crop

### PR DESCRIPTION
**Pull request by gauravahlawat81**
Fixes #106 

**Description**
Some creator links are opening an extension in the browser, it was because when there wasn't any creatorUrl, it was set to `#`, so instead of creating an anchor tag, I checked if there was a valid URL or not, if there wasn't, I created a paragraph tag in which Creator Name was displayed in plain text and not a hyperlink.

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: src/popup/infoPopupModule.js [hunk 1]
```diff
@@ -131,6 +131,15 @@ function getTumblrShareLink(sourceLink, imageLink) {
 }
 
 function getPopupCreatorChildNode(creatorUrl, creator) {
+  // This following code fixes the issue #106
+  // if the creatorURL doesn't exist, we do not create
+  // a anchor tag, instead we just create a paragraph tag
+  if (creatorUrl === '#') {
+    const paragraph = document.createElement('p');
+    paragraph.target = '__blank';
```
Comment (makkoncept) [inline src/popup/infoPopupModule.js]: This line does not do anything. Paragraph element does not have any attribute `target`. Please remove this.

**Review by makkoncept — CHANGES_REQUESTED** [review]
I have added a comment. Also, please uncheck the 4th checkbox in the checklist of the PR.

File: src/popup/infoPopupModule.js (hunk 1 above)
Comment (gauravahlawat81) [reply src/popup/infoPopupModule.js]: Oh yeah, sorry about that.

**Review by makkoncept — APPROVED** [review]
@gauravahlawat81 Looks good!

**Comment by gauravahlawat81** [comment]
Thanks, and sorry if I am still bad at writing meaningful commit messages. I will try to improve next time.

**Pull request merged**